controllers/argocd/statefulset_test.go

```diff
@@ -364,3 +364,10 @@ func Test_UpdateNodePlacementStateful(t *testing.T) {
 		t.Fatalf("updateNodePlacement failed, value of changed: %t", actualChange)
 	}
 }
+func Test_ContainsBrokenPod(t *testing.T) {
+	a := makeTestArgoCD()
+	ss := &appsv1.StatefulSet{}
+	if containsBrokenPod(ss, a) {
```

Comment:
we are passing an empty statefulset to find brokenPods ? Can you add more tests ?

Reply:
Yes, I'm still not sure how to test it properly because, the function fetches the podStatus, and since no pod exists it will return false always. So just added empty values.